mkosi/config.py

```diff
@@ -2325,7 +2346,12 @@ def load_config(args: argparse.Namespace) -> MkosiConfig:
     ):
         die("This unprivileged build configuration requires at least Linux v5.11")
 
-    return MkosiConfig.from_namespace(args)
+    config = MkosiConfig.from_namespace(args)
+    if config.want_uki() and not (shutil.which("ukify") or Path("/usr/lib/systemd/ukify").exists()):
+        die("'ukify' not found.",
+            hint="Bootable=no can be used to create a non-bootable image.")
```

Comment:
So I forgot about the tools tree, ukify can come from the tools tree but we only mount that later. So either we have to delay this check until any tools tree is mounted or we only have to do the check if there is no tools tree define here.

Reply:
I moved the check back to `check_inputs()`. This is simple and should be fairly robust. I don't want to implement  some complicated approach where the check is delayed.

Reply:
Oh, OK, now I grok what you mean. Updated again.